src/main/java/org/alliancegenome/curation_api/services/AlleleDiseaseAnnotationCrudService.java

```diff
@@ -37,4 +54,77 @@ protected void init() {
         return new ObjectResponse<AlleleDiseaseAnnotation>(alleleDiseaseAnnotationDAO.persist(dbEntity));
     }
 
+    @Transactional
+    public void runLoad(String taxonId, List<AlleleDiseaseAnnotationDTO> annotations) {
+        List<String> annotationIdsBefore = new ArrayList<>();
+        annotationIdsBefore.addAll(alleleDiseaseAnnotationDAO.findAllAnnotationIds(taxonId));
+        annotationIdsBefore.removeIf(Objects::isNull);
+        
+        log.debug("runLoad: Before: " + taxonId + " " + annotationIdsBefore.size());
+        List<String> annotationIdsAfter = new ArrayList<>();
+        ProcessDisplayHelper ph = new ProcessDisplayHelper(10000);
+        ph.startProcess("Disease Annotation Update " + taxonId, annotations.size());
```

Comment:
Might want to change this to display the type that is running (AGM)

Reply:
Done